spec/controllers/items_controller_spec.rb

```diff
@@ -220,6 +220,61 @@
     end
   end
 
+  describe '#rights' do
+    before do
+      allow(Dor::Services::Client).to receive(:object).with(pid).and_return(object_client)
+    end
+
+    let(:object_client) { instance_double(Dor::Services::Client::Object, find: cocina) }
+    let(:cocina) { instance_double(Cocina::Models::DRO, version: 1) }
+
+    context 'without an assigned apo' do
```

Comment:
All objects have an APO.  It is required.  This is not a case we would need to deal with.

Reply:
Is that guaranteed? One of the fixture objects doesn't have an APO (@apo is empty, not nil). Happy to remove if there isn't a reason we'd get there.

Reply:
The Ur-APO is the only object that is permitted to not have an APO.